git-secrets

```diff
@@ -205,8 +205,12 @@ process_output() {
     0)
       [ -z "${allowed}" ] && echo "${output}" >&2 && return 1
       # Determine with a negative grep if the found matches are allowed
-      echo "${output}" | GREP_OPTIONS= LC_ALL=C grep -Ev "${allowed}" >&2 \
-        && return 1 || return 0
+      #
+      # Clean the output in case of a trailing \r, \n (or combination of those)
+      matches=$(echo "${output}" | GREP_OPTIONS= LC_ALL=C grep -Ev "${allowed}")
+      cleaned_matches=$(echo "${matches}" | awk 'NR>1{print PREV} {PREV=$0} END{printf("%s",$0)}' | tr -d $'\r')
```

Comment:
`NR>1{print PREV}` prints the previous line (except the first time around)
`{PREV=$0}` will store the current line in `PREV`
`END{printf("%s",$0)}` prints the last line without the break